nx-dev/ui-markdoc/src/lib/tags/project-details.component.tsx

```diff
@@ -110,7 +111,7 @@ export function ProjectDetails({
       )}
       <div
         id="project-details-container"
-        className="not-prose overflow-y-auto"
+        className={twMerge('not-prose', height && 'overflow-y-auto')}
```

Comment:
There was a scrollbar being rendered for me. We don't need to add overflow unless the height property is set.